cardano-cli/src/Cardano/CLI/Types/Errors/TxCmdError.hs

```diff
@@ -48,7 +48,7 @@ data TxCmdError
   | TxCmdTxBodyError !TxBodyError
   | TxCmdNotImplemented !Text
   | TxCmdWitnessEraMismatch !AnyCardanoEra !AnyCardanoEra !WitnessFile
-  | TxCmdPolicyIdsMissing ![PolicyId]
+  | TxCmdPolicyIdsMissing ![PolicyId] ![PolicyId]
```

Comment:
Can you document what the two values stand for here? Since they have the same type, this declaration looks confusing without documentation.